example-playbooks/replace_node/replace_node.yml

```diff
@@ -24,6 +24,46 @@
       when: hostvars[new_node]['broadcast_address'] in scylla_seeds or new_node in scylla_seeds
       run_once: true
 
+    - name: Stop scylla-server on the node being replaced
+      block:
+        - name: Disable Scylla service
+          service:
+            name: scylla-server
+            enabled:  false
+          become: true
+
+        - name: Run nodetool drain
+          async_task:
+            shell: |
+              nodetool drain
+            alias: drain_scylla
+            async: 60
+            retries: 60
+            delay: 1
+          ignore_errors: true
+          register: _drain_scylla_output
+
+        - name: Stop Scylla service
+          async_task:
+            shell: |
+              systemctl stop scylla-server
+            alias: stop_scylla
+            async: 60
```

Comment:
We don't want to wait for the whole minute here. Here we want to wait, say 5 seconds and then kill the bastard (like you do below).

Reply:
Fixed